src/app/login/auth.service.ts

```diff
@@ -1,14 +1,16 @@
 import { HttpClient, HttpHeaders } from '@angular/common/http';
 import { Injectable } from '@angular/core';
-import { BehaviorSubject, Observable } from 'rxjs';
+import { BehaviorSubject, from, Observable } from 'rxjs';
 import { filter, map, tap } from 'rxjs/operators';
 import { environment } from '../../environments/environment';
 import { User } from '../model/user';
 
+//todo: improve the 2 behaviorSubject into one subject
```

Comment:
previously auth.service was handled by only one subject (behaviourSubject for User)..
- it looks like when using only one subject at the time of the id_token being stored and subject.next executed.. the catalog-count service "fetch list" is triggered.. so at the end more that 3 times this endpoint is called.. later when user object /v1/user is called.